test/ess-test-r-utils.el

```diff
@@ -98,6 +98,21 @@ inserted text."
           (ess-ask-for-ess-directory nil))
       (R "--vanilla"))))
 
+
+(defvar ess-r-test-proc-buf
+  (let* ((buf (run-ess-test-r-vanilla))
+         (proc (get-buffer-process buf)))
+    (ess-wait-for-process proc)
+    (with-current-buffer buf
+      (let ((inhibit-read-only t))
+        (erase-buffer)))
+    buf)
+  "Common process buffer for tests.")
```

Comment:
This creates a process whenever this file is loaded, right? I'm not sure that's the right approach; I'd like to be able to have the file load without side effects like that. (I'm also not sure of the behavior on byte compilation)

Reply:
Good point, we could generate it lazily with a closure instead.

Reply:
done!